packages/dev/core/src/Maths/math.vector.ts

```diff
@@ -6567,10 +6577,16 @@ export class Matrix implements Tensor<Tuple<Tuple<number, 4>, 4>>, IMatrixLike {
         return this;
     }
 
+    /**
+     * This method performs a component-by-component multiplication of the current matrix with the array of transmitted numbers.
+     * Use multiply or multiplyToRef for matrix multiplication.
+     * @param floats defines the array of numbers to multiply the matrix by
+     * @returns the current updated matrix
+     */
     public multiplyByFloats(...floats: Tuple<number, 16>): this {
         const m = this._m;
         for (let i = 0; i < 16; i++) {
-            m[i] = floats[i];
+            m[i] *= floats[i];

```

Comment:
wow, how long has that been in?

Reply:
Not very long, because this is a new method that came with the reorganization of the math classes.